app/src/main/report/components/MainMenu/MainMenu.tsx

```diff
@@ -1,19 +1,12 @@
 import * as React from "react";
 import { ReportingPageWithHeader } from "../ReportingPageWithHeader";
-import {reportStore} from "../../stores/ReportStore";
-import {doNothing} from "../../../shared/Helpers";
 import {ReportList} from "../Reports/ReportList";
 import {IPageWithParent} from "../../../shared/models/Breadcrumb";
 import { Page } from "../../../shared/components/PageWithHeader/Page";
 
-const menuStyles = require("./MainMenu.css");
+import "./MainMenu.scss";
 
 export class MainMenu extends ReportingPageWithHeader<undefined> {
-    load(props: undefined) {
-        return this.loadParent(props).then(() => {
-            return reportStore.fetchReports();
```

Comment:
So is this just not needed now because it happens in the child component? And bc this page doesn't have a parent?

Reply:
there is no need to load list of reports on details page

Reply:
Sorry I meant the `ReportList` component, child in the compositional sense!